srcpkgs/azote/template

```diff
@@ -22,6 +22,8 @@ post_install() {
 	vcopy "dist/${pkgname}.svg" usr/share/${pkgname}
 	vcopy "dist/${pkgname}-void.sh" usr/share/${pkgname}
 	vcopy "dist/${pkgname}.desktop" usr/share/applications
+	vlicense LICENSE
+	vlicense LICENSE-COLORTHIEF
```

Comment:
This looks like a BSD license.  You can add that to the license on line 12:

```
license="BSD-3-Clause, GPL-3.0-or-later"
```

Reply:
Yes, and it needs to be published, as I included the module directly into my python package. Thanks, I'll try what you wrote above.